README.md

```diff
@@ -1 +1,68 @@
-# thenextweb-clone
\ No newline at end of file
+# Project 2: Building with Responsive Design
+
+![screenshot](./Desktop.jpg)
+
+![screenshot](./Mobile.jpg)
```

Comment:
- [ ] Please add the proper description of the image you are presenting in Readme. You also need to add a Tablet version image here. When you make a tablet version of the webpage, Please add its screenshot here.